README.md

```diff
@@ -19,6 +19,30 @@ make test
 
 More complete documentation can be found in the [`docs`](./docs) directory.
 
+## Deployment
+### Pre-release
+1. Create a branch named "release-[VERSION]"
+2. Update `CHANGELOG.md` and `setup.py`
+3. Open a PR against the `master` branch
+4. Create and push the new tag
+5. Checkout the new tag before moving on to step #6
+6. Build wheels and create the tarball package
+
+   Hop over to the the `securedrop-debian-packaging` repo and follow the [build-a-package](https://github.com/freedomofpress/securedrop-debian-packaging/blob/master/README.md#build-a-package) instructions
+
+7. Push the package up to our PyPi mirror: https://pypi.org/simple
```

Comment:
So: I added a release guide in this repo in #71 which was moved to docs/development.rst by #73. tbh I think we should move everything in `docs/` into the README for discoverability since there's _really_ only one page of docs (I didn't even remember to look in there). The content of docs is not currently being rendered by readthedocs, and anything that should be rendered by readthedocs should just get merged to the main SecureDrop repo. Going to do that in a followup commit...